### PR TITLE
Charts branch

### PR DESCRIPTION
**Pull request by apoorvadhanala**
Added a pie chart to show music listening in different genres

@sidharthp please look at this when you can

### CODE REVIEW

File: portfolio/src/main/webapp/favorites.html [hunk 1]
```diff
@@ -1,9 +1,10 @@
-<!DOCTYPE html>
+ch<!DOCTYPE html>
```
Comment (sidharthp) [inline portfolio/src/main/webapp/favorites.html:1]: Did you mean to add `ch`?

File: portfolio/src/main/webapp/favorites.html [hunk 2]
```diff
@@ -46,6 +52,7 @@ <h1 class="heading-font">FAVORITE QUOTES</h1>
       <p class="body-font">Click to see one of my favorite quotes!</p>
       <button onclick="addRandomQuote()">Let's see it!</button>
       <div id="quote-container"></div>
+
```
Comment (sidharthp) [inline portfolio/src/main/webapp/favorites.html:55]: nit: You can remove this extra blank line.

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -59,9 +59,50 @@ function createListElement(task) {
...
+  "art exhibitions, and the Robert C. Williams Paper Museum. Near the Georgia Institute of Technology campus, old-school diners serve breakfast and burgers."
+  },
+  {
+      name: 'Georgia Aquarium',
+      lat: 33.7634,
+      long: -84.3951,
+      description: "Georgia Aquarium is a public aquarium in Atlanta, Georgia, United States. Georgia Aquarium is home to hundreds of species " + 
+  "and thousands of animals across its seven major galleries, all of which reside in more than 10 million US gallons of fresh and salt water."
+  },
+];
```
Comment (sidharthp) [inline portfolio/src/main/webapp/script.js:99]: nit: Keep the variable definitions before any of the functions.

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -59,9 +59,50 @@ function createListElement(task) {
   return liElement;
 }
 
+var map;
```
Comment (sidharthp) [inline portfolio/src/main/webapp/script.js:62]: This doesn't need to be a global variable, right?

File: portfolio/src/main/webapp/script.js [hunk 5]
```diff
@@ -147,25 +188,55 @@ function createMap() {
...
+    'width':700,
+    'height':600,
+    pieHole: 0.4,
+    colors: ['#E082AA','#E0BEF7', '#F7F2BE','#82E0D7', '#CDECE8']
+  };
+
+  const chart = new google.visualization.PieChart(
+    document.getElementById('chart-container'));
+  chart.draw(data, options);
+
```
Comment (sidharthp) [inline portfolio/src/main/webapp/script.js:239]: nit: Remove extra blank line.

File: portfolio/src/main/webapp/script.js [hunk 6]
```diff
@@ -147,25 +188,55 @@ function createMap() {
...
+  }
```
Comment (sidharthp) [inline portfolio/src/main/webapp/script.js:240]: nit: remove 2 spaces from the beginning.

File: portfolio/src/main/webapp/script.js [hunk 7]
```diff
@@ -147,25 +188,55 @@ function createMap() {
...
+
```
Comment (sidharthp) [inline portfolio/src/main/webapp/script.js:241]: nit: Remove extra blank line.

File: portfolio/src/main/webapp/style.css [hunk 8]
```diff
@@ -144,3 +144,9 @@ body, html {
   margin-right:auto;
   border: thick solid LIGHTCORAL;
 }
+
+#chart-container{
+    display: inline-block;
+    margine: 0 auto;
+
+}
```
Comment (sidharthp) [inline portfolio/src/main/webapp/style.css:152]: nit: Add blank line at the end of the file.

File: portfolio/src/main/webapp/style.css [hunk 9]
```diff
@@ -144,3 +144,9 @@ body, html {
   margin-right:auto;
   border: thick solid LIGHTCORAL;
 }
+
+#chart-container{
+    display: inline-block;
+    margine: 0 auto;
+
```
Comment (sidharthp) [inline portfolio/src/main/webapp/style.css:151]: nit: Remove blank line.

File: portfolio/src/main/webapp/favorites.html (hunk 1 above)
Comment (apoorvadhanala) [reply portfolio/src/main/webapp/favorites.html:1]: got rid of it!

File: portfolio/src/main/webapp/script.js (hunk 4 above)
Comment (apoorvadhanala) [reply portfolio/src/main/webapp/script.js:62]: Jenny told me to so that it's possible to reference in any scope level

**Pull request merged**